### PR TITLE
Improve align & frmt_combine docs

### LINKED ISSUE #54: Add Error message to frmt_combine when no matching label isn't there
image
Needs to either yell at me or have 10 rows

### LINKED ISSUE #91: Improve the documentation reporting for col_align of non-value columns

### PR DESCRIPTION
**Pull request by bzkrouse**
resolves #54 
resolves #91

### CODE REVIEW

File: tests/testthat/test-apply_frmt.R [hunk 1]
```diff
@@ -340,6 +340,62 @@ test_that("applying frmt_combine - 3x", {
...
+
+  sample_df_frmted <- suppressWarnings(apply_frmt.frmt_combine(
+    frmt_def = sample_frmt,
+    .data = sample_df,
+    values = quo(x),
+    param = quo(y),
+    column = vars(col),
+    label = quo(lab),
+    group = vars(group)
+  ))
```
Comment (thebioengineer) [inline tests/testthat/test-apply_frmt.R:382]: You can assign outputs from expect_warning, so you don't need to rerun the code and suppress warnings.

File: R/apply_frmt_methods.R [hunk 2]
```diff
@@ -200,6 +200,17 @@ apply_frmt.frmt_combine <- function(frmt_def, .data, values, mock = FALSE, param
...
 
+  # check that pivot_wider resulted in a reduction of rows, which indicates that at least
+  #  1 row will successfully have a frmt_combine in it
+  if (nrow(.tmp_data_wide)==nrow(.tmp_data)){
+    id_cols <- .tmp_data_wide %>% select(!!!column, !!label, !!!group)
+    warning(paste0("Unable to apply `frmt_combine` due to uniqueness of column/row identifiers. Params that are to be combined need to have matching values across: ",
+                   paste(names(id_cols), collapse = ", "),
+                   ". Current values:\n",
+                   paste(capture.output(id_cols %>% as.data.frame), collapse = "\n")))
+  }
```
Comment (thebioengineer) [inline R/apply_frmt_methods.R:211]: Should we add the params as well?

**Review by thebioengineer — CHANGES_REQUESTED** [review]
Thanks for getting this together. Let me know if my comments don't make sense!

File: tests/testthat/test-apply_frmt.R [hunk 3]
```diff
@@ -340,6 +340,62 @@ test_that("applying frmt_combine - 3x", {
...
+  expect_warning(
+    apply_frmt.frmt_combine(
+    frmt_def = sample_frmt,
+    .data = sample_df,
+    values = quo(x),
+    param = quo(y),
+    column = vars(col),
+    label = quo(lab),
+    group = vars(group)
+  ))
```
[the remark this answers is not in the document]
Comment (bzkrouse) [reply tests/testthat/test-apply_frmt.R]: Sure, I added the beginning of the message as I was having issues with the rest. LMK if you think thats sufficient.

**Review by thebioengineer — APPROVED** [review]

**Comment by thebioengineer** [comment]
Looks great. Thanks!

**Pull request merged**